### PR TITLE
NIAD-348: MESH API authorization tests

### PR DESCRIPTION
**Pull request by grzegorz-robak**

### CODE REVIEW

File: src/intTest/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshAuthorizationIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    }
+
+    @SneakyThrows
+    private SSLContext getFakeMeshSSLContext() {
+        KeyStore ks = EnvKeyStore.createFromPEMStrings(
+                new String(Files.readAllBytes(fakeMeshKey.getFile().toPath())),
+                new String(Files.readAllBytes(fakeMeshCert.getFile().toPath())),
+                "")
+            .keyStore();
+        return SSLContexts.custom().loadKeyMaterial(ks, "".toCharArray())
```
Comment (jaroslaw-wojciechowski) [inline src/intTest/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshAuthorizationIntegrationTest.java]: static String password = StringUtils.EMPTY ?

File: src/intTest/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshAuthorizationIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            .loadTrustMaterial(TrustSelfSignedStrategy.INSTANCE)
+            .build();
+    }
+
+    @SneakyThrows
+    private SSLContext getFakeMeshSSLContext() {
+        KeyStore ks = EnvKeyStore.createFromPEMStrings(
+                new String(Files.readAllBytes(fakeMeshKey.getFile().toPath())),
+                new String(Files.readAllBytes(fakeMeshCert.getFile().toPath())),
+                "")
```
Comment (bartek-sarul) [inline src/intTest/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshAuthorizationIntegrationTest.java]: Use `StringUtils.EMPTY` to explicitly say you wanted an empty string here

**Comment by bartek-sarul** [comment]
Minor change request + build is red

**Review by bartek-sarul — CHANGES_REQUESTED** [review]
Minor change request + build is red

**Review by jaroslaw-wojciechowski — APPROVED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Review by Alicja-Kainos — APPROVED** [review]

**Review by bartek-sarul — APPROVED** [review]

**Pull request merged**